generic/tests/iozone_windows.py

```diff
@@ -117,8 +118,15 @@ def check_gpt_labletype(disk_index):
     if params.get("format_disk", "no") == "yes":
         error_context.context("Format disk", logging.info)
         for index, letter, fstype in zip(disk_indexes, disk_letters, disk_fstypes):
-            utils_misc.format_windows_disk(session, index, letter, fstype=fstype,
-                                           labletype=labletype)
+            orig_letters = utils_disk.get_drive_letters(session, index)
```

Comment:
@menli820 using the function \"utils_disk.configure_empty_windows_disk()\", it may work well without avocado-vt's update. the usage you can refer to case multi_disk.py. I think if we also hit this issue when using utils_disk.configure_empty_windows_disk, then we update avocado-vt codes.